sentry_sdk/integrations/aiohttp.py

```diff
@@ -286,7 +353,60 @@ def aiohttp_processor(
 
         return event
 
-    return aiohttp_processor
+    return aiohttp_server_processor
+
+
+def _make_client_processor(trace_config_ctx, response, response_content):
+    # type: (SimpleNamespace, Response, Optional[Dict[str, Any]]) -> EventProcessor
+    def aiohttp_client_processor(
+        event,  # type: Dict[str, Any]
+        hint,  # type: Dict[str, Tuple[type, BaseException, Any]]
+    ):
+        # type: (...) -> Dict[str, Any]
+        with capture_internal_exceptions():
+            request_info = event.setdefault("request", {})
+
+            parsed_url = parse_url(str(response.url), sanitize=False)
+            request_info["url"] = parsed_url.url
+            request_info["query_string"] = parsed_url.query
+            request_info["method"] = response.method
+
+            if getattr(trace_config_ctx, "request_headers", None):
+                request_info["headers"] = _filter_headers(
+                    dict(trace_config_ctx.request_headers)
+                )
+            if getattr(trace_config_ctx, "request_body", None):
+                request_info["data"] = trace_config_ctx.request_body
```

Comment:
Guard with with `_should_send_default_pii()`. There is probably all kinds of personal data in the graphql request.

Reply:
thx, done!